server.py

```diff
@@ -32,6 +33,47 @@ def get_shopcarts(id):
 
     return jsonify(message), rc
 
+######################################################################
+# Create a Shopcart
+######################################################################
+@app.route('/shopcarts/', methods=['POST'])
+def create_shopcart():
+    """
```

Comment:
Too much logic for a controller IMO. Also gets deeply nested which we should try to avoid.

Reply:
There is no point in undoing this. It actually makes sense. @thoughts1053 you are making a comment as a Product Manager in a place where it does not make sense. Here you should comment as a developer who is looking that the code works.

We can comment this on future meetings but I don't see a problem here.

The program does what the issue tells it should do and there is no point of changing for what you are asking.

What do you guys think? @RegulusXiang @radhikamattoo

Reply:
After giving it a thought I actually agree, @thoughts1053 . 
Take a look know.